build_tools/benchmarks/common/benchmark_definition.py

```diff
@@ -208,28 +227,27 @@ def __str__(self):
     params = ", ".join(params)
     return f"{self.platform_type.value} device <{params}>"
 
-  def get_iree_cpu_arch_name(self) -> str:
-    arch = CPU_ABI_TO_TARGET_ARCH_MAP.get(self.cpu_abi.lower())
-    if not arch:
-      raise ValueError(f"Unrecognized CPU ABI: '{self.cpu_abi}'; "
-                       "need to update the map")
-
+  def get_iree_cpu_arch_name(self,
```

Comment:
Do we actually want to transform into the string here? This function seems like it would be more useful if it got the object itself. Programming with objects and only converting to strings when necessary is generally safer. Looking at the usage, at least in these files, explicit string conversion is already being performed. I see that in the legacy case we have to return a string, so that makes this a bit less appealing. You could consider making this `Optional[Union[str, common_definitions.DeviceArchitecture]]` or leaving a TODO to raise up the level of abstraction of this function once the legacy path is removed

Reply:
I think pytype will not be happy to handle union return value. So I'd prefer to return string here. Added a TODO to make sure we switch to enum type later.